### ISSUE TITLE
Changes to app-deploy.yaml are not persisted during appsody build

### ISSUE THREAD

**Issue by kylegc** [appsody/appsody#644]
For now, we have the app-deploy.yaml generated on `appsody build` if it doesn't exist, and updated if it does exist. The updates should be limited to the labels, annotations, and applicationImage.

Two examples where custom changes get wiped out on `appsocy build`:
- added a custom label
- added a section to the `service` section
```    consumers:
    - serviceName: "appsody-backend"
      namespace: default
      category: openapi```

**Comment by EnriqueL8** [comment]
It could be a limitation of the Operator: https://github.com/appsody/appsody-operator/blob/master/doc/user-guide.md#custom-resource-definition-crd

**Comment by EnriqueL8** [comment]
Fixed in #656
